Gradle.md

```diff
@@ -94,6 +94,15 @@ The scripts return with an exit code that depends on the outcome of running a gi
 | 3         | The build was not fully cacheable for the given task graph and `--fail-if-not-fully-cacheable` was set |
 | 100       | An unclassified, fatal error happened while running the experiment                                     |
 
+## Verifying the setup
+
+You can verify that the script can properly interact with your Develocity server. In the example below, the script will configure 
+the local builds to publish their build scans to a Develocity server reachable at develocity.example.io.
+
+```bash
+./01-validate-incremental-building.sh -t assemble -r https://github.com/gradle/gradle-build-scan-quickstart -e -s https://develocity.example.io
```

Comment:
Just running `help` is enough to verify connectivity. That would avoid `0.5s` of unnecessary waiting time. 

`assemble` is fine too though. 😉